packages/driver/src/cypress/error_utils.ts

```diff
@@ -17,9 +15,9 @@ const crossOriginScriptRe = /^script error/i
 
 if (!Error.captureStackTrace) {
   Error.captureStackTrace = (err, fn) => {
-    const stack = (new Error()).stack
+    const stack = (new Error()).stack;
```

Comment:
```suggestion
    const stack = (new Error()).stack
```

Reply:
`;` is necessary here because JavaScript tries to translate `.stack\n\n(err as Error)` as a function call.